File: codewars-js-ts/fizzBuzz.js

const fizzBuzz = (number) =>{
  const fizzBuzzArr = []
  for(let i = 1; i <= number; i ++) {
    if(i % 3 === 0 && i % 5 === 0) {
      fizzBuzzArr.push('FizzBuzz')
    }
    else if(i % 3 === 0) {
      fizzBuzzArr.push('Fizz')
    }
    else if ( i % 5 === 0) {
      fizzBuzzArr.push('Buzz')
    }

    else {
      fizzBuzzArr.push(i)
    }

  }


  return fizzBuzzArr
}

const result = fizzBuzz(10)

result


module.exports = fizzBuzz

File: codewars-python/fizz_buzz.py
def fizzbuzz(number):
  arr = []

  for i in range(1,number):
    if i % 5 == 0  and i % 3 == 0:
      arr.append("FizzBuzz")
    elif i % 3 == 0:
      arr.append("Fizz")
    elif i % 5 == 0:
      arr.append("Buzz")
    else:
      arr.append(i)

  return arr





